ci/kind/test-e2e-kind.sh

```diff
@@ -125,6 +131,10 @@ function run_test {
   current_mode=$1
   args=$2
 
+  if [[ "$ipfamily" == "ipv6" ]]; then
```

Comment:
can you "sanitize" the value of `ipfamily` here and error out if it's not valid?